### ISSUE TITLE
DOC/STY: multiple definitions of instrument properties

### ISSUE THREAD

**Issue by jklenzing** [pysat/pysat#651]
Warnings during build of the docs:
```
pysat/pysat/_instrument.py:docstring of pysat.Instrument.bounds:1: WARNING: duplicate object description of pysat.Instrument.bounds, other instance in api, use :noindex: for one of them
pysat/pysat/_instrument.py:docstring of pysat.Instrument.date:1: WARNING: duplicate object description of pysat.Instrument.date, other instance in api, use :noindex: for one of them
```
This results from multiple definitions of `bounds` and `date` using `@property` and `@bounds.setter` / `@date.setter` and the usage of automodule.
Possible solutions:
- Adjust the usage of automodule to specify which is used with :noindex:
- rewrite the instrument properites using a single definition?

**Comment by rstoneback** [comment]
I think we only need to add the :noindex:.

**Comment by jklenzing** [comment]
I've yet to find a general examples, but many projects recommend only including docstrings in the `getter` if both `getter` and `setter` exist.

https://developer.lsst.io/v/DM-15183/python/numpydoc.html

**Comment by jklenzing** [comment]
Just saw this is how it's handled in `Instrument`.  Will update for `Constellation` in my current branch.

**Comment by jklenzing** [comment]
Fixed for constellation in #879.
